src/components/Button/ButtonContent/styled.ts

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components'
+
+// @TODO: Solve how to change svg color
+export const ButtonIcon = styled.img`
+	width: 2.5em;
+`
+
+export const ButtonTitle = styled.span`
+	font-size: 1em;
+	text-transform: uppercase;
+	margin: 0.2em 0.5em;
+`
+
+export const ButtonSubtitle = styled.p`
+	font-size: 0.5em;
+	margin: 0;
+	padding: 0;
+`
+
+// @NOTE: The background color is better when it is transparent, but I don't know how to do it without rgba and opacity.
+export const ButtonCenter = styled.div`
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	flex-direction: column;
+	font-family: Verdana, Geneva, Tahoma, sans-serif;
+	padding: 0.6em 1em;
+	cursor: pointer;
+	border-radius: 0.8em;
+	background-color: ${({ theme }) => theme.bg.secondary};
+	color: ${({ theme }) => theme.text.secondary};
+	&:hover {
+		background-color: ${({ theme }) => theme.bg.secondary};
+		color: ${({ theme }) => theme.text.link};
+	}
+`
+
+// @NOTE: ButtonLine may have other colors. We can have two different buttons.
+// @NOTE: Topic for discussion.
+export const ButtonLine = styled.div`
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	font-family: Verdana, Geneva, Tahoma, sans-serif;
+	padding: 0.6em 1em;
+	cursor: pointer;
+	border-radius: 0.8em;
+	background-color: rgba(255, 0, 0, 0.3);
+	color: yellow;
```

Comment:
Pozor, barvy jsou definované pro téma. Aby se měnily automaticky, pokud bychom změnili barvy celému téma (a podle Dark/Light modu) musí být definované jako:

`color: ${({ theme }) => theme.text.secondary};` (tohle je přímo ta naše žlutá)

Totéž všude jinde, kde jsou použité barvy z naší color palette.

Reply:
Neboj vím a dávám pozor 🙂 Já jsem záměrně používala zatím jen button s layout center, který víme, že bude použit u navigace a víme přibližně, jak bude vypadat. U button s layout line, který by se podle mě hodil na přihlášení atd. mě napadlo, že by mohl mít jiný barevný styl, a tak jsem prozatím mu nedávala žádné významné barvy 🙂 Jestli to mám změnit, případně na co, stačí říct a provedu.